source/gridfs/gridfs-spec.rst

```diff
@@ -727,47 +727,58 @@ Generic Find on Files Collection
 .. code:: javascript
 
   class GridFSFindOptions {
-  
+
+    /**
+     * Enables writing to temporary files on the server. When set to true, the server
+     * can write temporary data to disk while executing the find operation on the files collection.
+     *
+     * This option is sent only if the caller explicitly provides a value. The default
+     * is to not send a value.
+     *
+     * @see https://docs.mongodb.com/manual/reference/command/find/
+     */
+    allowDiskUse: Optional<Boolean>;
```

Comment:
Is this change actually necessary?

IIRC, `Bucket::find()` is just a proxy for a `find()` operation on the `fs.files` collection and opens a cursor on the raw documents (nothing to do with opening download streams).

I don't recall why we defined GridFSFindOptions separately instead of supporting FindOptions in the CRUD spec, but the underlying operation should be the same. GridFSFindOptions was also never updated to support options such as `collation`. Was that intentional or just an oversight?

I might propose removing this and just creating a new SPEC ticket to consolidate and just reference FindOptions  and the CRUD spec unless there's good reason to keep them separate.

Reply:
Opened https://jira.mongodb.org/browse/SPEC-1549 to track this.